fs/operations/operations.go

```diff
@@ -601,25 +595,40 @@ func Same(fdst, fsrc fs.Info) bool {
 	return SameConfig(fdst, fsrc) && strings.Trim(fdst.Root(), "/") == strings.Trim(fsrc.Root(), "/")
 }
 
+// FixRoot returns the Root with a trailing / if not empty
```

Comment:
I don't think this function needs to be exported does it?  so it should be `fixRoot`.

Also the comment is now wrong!